app/gui2/src/stores/graph/graphDatabase.ts

```diff
@@ -1,3 +1,4 @@
+import type { Node, NodeAstData, NodeMetadataData } from '@/stores/graph'
```

Comment:
We have now cyclic dependency between modules; I'd prefer to avoid that at all cost. Please bring `Node` structure back to this file.